mp/src/game/client/viewpostprocess.cpp

```diff
@@ -1609,6 +1618,43 @@ void DumpTGAofRenderTarget( const int width, const int height, const char *pFile
 
 static bool s_bScreenEffectTextureIsUpdated = false;
 
+// WARNING: This function sets rendertarget and viewport. Save and restore is left to the caller.
+static void DownsampleFBQuarterSize( IMatRenderContext *pRenderContext, int nSrcWidth, int nSrcHeight, ITexture* pDest,
+									 bool bFloatHDR = false )
+{
+	Assert( pRenderContext );
+	Assert( pDest );
+
+	IMaterial *downsample_mat = materials->FindMaterial( bFloatHDR ? "dev/downsample" : "dev/downsample_non_hdr", TEXTURE_GROUP_OTHER, true );
+
+	// *Everything* in here relies on the small RTs being exactly 1/4 the full FB res
+	Assert( pDest->GetActualWidth()  == nSrcWidth  / 4 );
+	Assert( pDest->GetActualHeight() == nSrcHeight / 4 );
+		
+	/*
+	bool bFound;
+	IMaterialVar *pbloomexpvar = downsample_mat->FindVar( "$bloomexp", &bFound, false );
+	if ( bFound )
+	{
+		pbloomexpvar->SetFloatValue( g_flBloomExponent );
+	}
+
+	IMaterialVar *pbloomsaturationvar = downsample_mat->FindVar( "$bloomsaturation", &bFound, false );
+	if ( bFound )
+	{
+		pbloomsaturationvar->SetFloatValue( g_flBloomSaturation );
+	}
+	*/
+
+	// downsample fb to rt0
+	SetRenderTargetAndViewPort( pDest );
+	// note the -2's below. Thats because we are downsampling on each axis and the shader
+	// accesses pixels on both sides of the source coord
+	pRenderContext->DrawScreenSpaceRectangle(	downsample_mat, 0, 0, nSrcWidth/4, nSrcHeight/4,
+												0, 0, nSrcWidth-2, nSrcHeight-2,
+												nSrcWidth, nSrcHeight );
```

Comment:
Spacing

Reply:
All the draw calls in that file (and others) are spaced like that, so I believe this should be fixed in a separate formatting PR instead.